notebook/tests/selenium/test_empty_arrows_keys.py

```diff
@@ -0,0 +1,13 @@
+# selenium test version for 'empty_arrow_keys.js'
+
+
+def remove_cells(notebook):
+    for i in notebook.cells:
+        notebook.delete_cell(notebook.index(i))
+
+def test_empty_arrows_keys(notebook):
+    # delete all the cells
+    notebook.trigger_keydown('up')
+    notebook.trigger_keydown('down')
```

Comment:
Let's get rid of the up/down, since they're not really testing anything, rename the function to `test_delete_all_cells`, and maybe move it into `test_deletecell.py`